doc/man3/EVP_CIPHER_CTX_get_original_iv.pod

```diff
@@ -47,7 +47,7 @@ different return type for the pointer.
 =head1 RETURN VALUES
 
 EVP_CIPHER_CTX_get_original_iv() and EVP_CIPHER_CTX_get_updated_iv() return 1
-on success and 0 on failure.
+on success and <=0 on failure.
```

Comment:
I would actually describe this as a bug in the code. We only return 0 or 1 virtually everywhere except in one location (in the case where an EVP_CIPHER_CTX with a legacy EVP_CIPHER is being passed)